src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -80,17 +81,37 @@ Ensure that you are able to run Docker without using sudo directly with the
 sudo usermod -aG docker $USER
 exec newgrp docker
 \`\`\`
-3. Install the dockstore command-line program and add it to the path.
+
+#### Part 1 for MacOS - Install dependencies
+1. Install Java 11. Download the JDK from [AdoptOpenJDK](https://adoptopenjdk.net/index.html). Choose the MacOS Compressed Archive Version.
```

Comment:
I install the JDK with HomeBrew. If you don't want to write that up in detail, I'd at least mention is as an alternative.

Reply:
Gave some instructions and also realized I left the wrong link in from this comment